recipes/keyrings.alt/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "keyrings.alt" %}
```

Comment:
The name is the same as the name of the corresponding pypi package (https://pypi.python.org/pypi/keyrings.alt/1.1.1). I would prefer that the names match as having different names for the same package using different package manager seems counter intuitive.